databricks/koalas/tests/test_series.py

```diff
@@ -386,3 +386,13 @@ def test_map(self):
         self.assertEqual(
             repr(kser.map(d)),
             repr(pser.map(d).rename(0)))
+
+    def test_pandas_wraps_and_isna(self):
```

Comment:
can you add a line of comment explaining what this test is doing?

Reply:
yup